html-relative-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Contact Us</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href ="../index.html">Home Page</a></li>
+    <li><a href ="../about-me/about-me.html">About Me</a></li>
+    <li><a href ="../projects/">Projects</a></li>
+    <li><a href ="../contact-us/contact-us.html">Contact Us</a></li>
```

Comment:
On this line, you are exiting the current directory just to go right back into it. Instead, you can just state the name of the file you need directly since it's in the same directory this path is being built from.